pkg/api/peer_req_resp.go

```diff
@@ -18,9 +18,17 @@ type PeerAddReq struct {
 	MetaData  map[string]string `json:"metadata"`
 }
 
+// MetaDataEditReq represents an incoming request to edit metadata of peer
+type MetaDataEditReq struct {
```

Comment:
This should be `PeerEditReq`. The same API can be used to update/edit peer later.